lib/fibers_stack/SFibersStackHitFinderPar.cc

```diff
@@ -24,14 +24,64 @@ A container for Fibers Stack Calibrator parameters
 \sa SPar
 
 */
-
+/**
+ * Number of parameters for the resolution fit, fixed to 3
+ */
+Int_t numOfResPars = 3;
```

Comment:
As I cannot test your branch from your fork, I am wondering whether you set `fResPos` and `fResEne` to have actually seize of `numOfResPars`. It must be as, while the size is set in the `getParams`, if user creates the category and directly tries to call `getPosRes(2)` probably will have error due to incorrect size.

In the constructor call:
```c++
fPosRes.Set(numOfResPars);
fEneRes.Set(numOfResPars);
```

Please also take `.clang-format` from the master branch (was updated) and reformat the code using this shell command:
```bash
find lib dst \( -name "*.h" -or -name "*.cc" \) ! -name "Linkdef.h" -exec clang-format --style=file -i {} \;
```